changelog.d/374.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Correct Towncrier's config entries in the :file:`pyproject.toml` file.

```

Comment:
This file was integrated as part of PR #375 and should _not_ appear in this branch. You should remove this file and rebase your branch. Possibily you need to resolve conflicts.

Reply:
I agree. Will do.